sources/packages/web/package.json

```diff
@@ -34,7 +34,7 @@
     "vue": "^3.2.31",
     "vue-class-component": "^8.0.0-0",
     "vue-router": "^4.3.2",
-    "vuetify": "^3.4.2",
+    "vuetify": "^3.7.2",
```

Comment:
Apparently, this upgrade fixes an issue with the pagination where there was no way to remove the "All" options that cause an API error right now.

![image](https://github.com/user-attachments/assets/6363ba10-529c-427e-bbd9-8caae5e90410)

Can you please check if the below configuration would allow removing the "All" from the options?
https://vuetifyjs.com/en/api/v-data-table/#props-items-per-page-options

Reply:
Fixed.